dev-tools/src/main/resources/ant/integration-tests.xml

```diff
@@ -37,8 +37,25 @@
     </sequential>
   </macrodef>
 
+  <!-- extracts PID from file -->
+  <macrodef name="extract-pid">
+      <attribute name="property"/>
```

Comment:
The spacing was intentional. attributes are the parameters that the macro takes. but this is all XML, so it can be hard to see, unless you do extra indentation.

This has always been the defacto style for lucene at least. Anything else seems crazy?